tools/topology/sof-tgl-rt711-rt1308.m4

```diff
@@ -74,6 +84,10 @@ ifdef(`EXT_AMP', `
 # PCM8 ---> volume <---- iDisp4
 # PCM10 <----volume <---- DMIC01
 # PCM11 <----volume <---- DMIC16k
+ifelse(PLATFORM, `adl', `
+# PCM13 ----> passthrough ----> SSP2 BT playback
+# PCM14 <---- passthrough <---- SSP2 BT capture
+', `')
```

Comment:
So as this is the PCM (FOO_PCM_BAR_ID) numbering, the documentation should be:
```
# PCM14 ----> passthrough ----> SSP2 BT playback
# PCM14 <---- passthrough <---- SSP2 BT capture
```

Reply:
Ack, I confused myself to say the least :) Will send update, thanks @plbossart and @kv2019i